ansible_collections/arista/avd/roles/eos_designs/templates/facts/switch/switch.j2

```diff
@@ -23,6 +23,15 @@ switch:
   platform_settings: {{ platform_settings | selectattr("platforms", "arista.avd.contains", switch_platform) | first | arista.avd.default(
                         platform_settings | selectattr("platforms", "arista.avd.contains", "default") | first) | to_json() }}
 
+{# switch.ignore_tcam_system_profiles #}
+{% if switch.vtep is arista.avd.defined(true) %}
+{%     set profiles = switch_data.combined.ignore_tcam_system_profiles | arista.avd.default([]) %}
+{% else %}
+{#     To ignore vxlan-routing profile by default on all non-vtep devices #}
```

Comment:
We can't change this default behavior until next major release. So I suggest that you keep line 26,28,33 and copy the rest to a todo note on our 4.0 project.
This way the project requesting this can add the knob they need for spines.
Please also rename the temp variable from`profiles` to `switch_ignore_tcam_system_profiles`. Thanks.

Reply:
I got your point. So you want the default behavior later in 4.0. Will make the changes accordingly.